.github/workflows/continuous-integration.yml

```diff
@@ -15,7 +15,6 @@ jobs:
         operating-system:
           - ubuntu-latest
         php-version:
-          - '5.5'
```

Comment:
Why do you drop 5.5?

Reply:
By mistaken. Fixed with 60b1367145da812038fee7e9f44998a0eaa27afc